src/model/index.js

```diff
@@ -14,8 +14,8 @@ Model.prototype.getData = (req, callback) => {
 
   db.data.getGeometryColumnName(schema, table)
     .then(result => {
-      const geom = result.f_geometry_column
-      const srid = result.srid
+      const geom = result?.f_geometry_column || 'null'
+      const srid = result?.srid || 'null'
```

Comment:
What is your intention with these values?  I have refactored this module on the `develop` branch and plan to update with something like the following: 

```javascript
const result = await db.data.getGeometryColumnName(schema, table);

if (!result) {
  throw new Error('Invalid result from getGeometryColumnName');
}
```

To error out gracefully.

Reply:
> ESRI Feature Services can support tables, which have no geometry. This
> change keeps the provider from erroring when reading postgres tables
> that have no geometry but can still be served as a Feature Layer

My intention is to be able to serve a table (no geometry) in a [feature service](https://developers.arcgis.com/rest/services-reference/enterprise/layer-table.htm). We have a use case where we are using Koop to create a feature service with tables for our Survey123 app to query and dynamically populate choices in the survey. When doing this initially I ran into exceptions because the postgres table had no columns of type `geometry`. The workaround I found was to set these to `null` and then Koop served a table as expected. Erroring gracefully here is still erroring when this is a case that Koop supports.